.gitattributes

```diff
@@ -0,0 +1,5 @@
+# This file is used by unit tests and data I/O testing the CRLF/LF
+#  mode of the checked out working tree is important to ensure the unit
+#  test passes.
+hibernate-core/src/test/resources/org/hibernate/test/fileimport/multi-line-statements.sql eol=lf text
```

Comment:
Would it be better to try and use a wildcard here?  Maybe something like the following?
`**/test/resources/**/*.sql eol-lf text`

If I remember correctly, git isn't able to automatically filter out binary files, so explicit file extensions are needed.  .sql is a good start.